packages/grid/x-data-grid-premium/src/tests/columns.DataGridPremium.test.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
```

Comment:
Off topic to this PR:
It could certainly be better to also move/resize the aggregation footer item with the resize (which happens on mouseout right now)
![aggregation](https://github.com/mui/mui-x/assets/12609561/208ceb5e-2c97-4ef2-90e9-884fe33fcbed)

Reply:
Yes, it's in a separate PR https://github.com/mui/mui-x/pull/10229